Build completely broken: String index out of range: 0

Even a simple `./gradlew clean` fails with this error:

```
➜  ./gradlew clean

FAILURE: Build failed with an exception.

* What went wrong:
String index out of range: 0
```

--- a/kotlin/gradle.properties
+++ b/kotlin/gradle.properties
@@ -20,13 +20,7 @@ android.useAndroidX=true
 #android.debug.obsoleteApi=true
 
 GROUP=com.squareup.workflow
-<<<<<<< HEAD
 VERSION_NAME=0.24.0-SNAPSHOT
-||||||| 93d98f6c
-VERSION_NAME=0.23.1-SNAPSHOT
-=======
-VERSION_NAME=0.23.2-SNAPSHOT
->>>>>>> release-v0.23.x
 
 POM_DESCRIPTION=Reactive workflows
 

